cmake/mod_bundled_packages_json.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# This script removes the given package entries in the bundled_packages JSON files. This is used by the PluginInstall
+# CMake module.
+#
+
+import argparse
+import collections
+import json
+import os
+import sys
+
+
+def find_json_files(work_dir: str) -> list:
+    """
```

Comment:
You can use docstrings rather than doxygen as long as you are writing non- core cura script?
I am obviously a big proponent of docstrings, just wanted to make sure what is allowed by the documentation police

Reply:
I don't know. I thought we still want to move to docstrings because we can't get arguments and types from doxygen.

Reply:
Officially we're on Doxygen, so that's what we need to use (because ya know, that's what the codestyle says)

Reply:
Converted to doxygen style in 7b9ababc1